projects/packages/paypal-payments/package.json

```
@@ -15,11 +15,48 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"build": "echo 'Not implemented.'",
+		"build": "pnpm run clean && webpack --config ./webpack.config.blocks.js",
 		"build-js": "echo 'Not implemented.'",
 		"build-production": "echo 'Not implemented.'",
 		"build-production-js": "echo 'Not implemented.'",
```

You can drop `build-js` and `build-production-js` if you're not going to use them. OTOH, you are calling `build-production` from composer.json, so that one should be implemented. It might be as simple as
```suggestion
		"build-production": "NODE_ENV=production BABEL_ENV=production pnpm build",
```

---

Thanks for taking a look! 

> You can drop build-js and build-production-js if you're not going to use them.

I'll go ahead and remove them. If we do end up needing them, it will be easy to re-add them.

> OTOH, you are calling build-production from composer.json, so that one should be implemented. It might be as simple as ...

Yes, thanks for flagging it. I'll get that updated. Based on what I saw in the forms package, your suggested command looks like it should be correct.

---

Addressed in [0735334](https://github.com/Automattic/jetpack/pull/43413/commits/07353346486079ae3752e3b1af07481526b4629f).